messages/list.json

```diff
@@ -1,3 +1,4 @@
 {
-  "description": "list auth connection information"
+  "description": "list auth connection information",
+  "noResultsFound": "No results found"
```

Comment:
nit: do you think we should provide more here? `No authorized environments. Run an auth command get started!` Maybe run it by Juliet.